src/viper/trans.ml

```diff
@@ -121,11 +156,17 @@ let rec extract_concurrency (seq : seqn) : stmt' list * seqn =
   let conc, stmts = List.fold_left extr ([], []) stmts in
   rev conc, { seq with it = fst seq.it, rev stmts }
 
+let prelude =
+  let (!!) p = !!! Source.no_region p in
+  [ !! (ImportI (!! "../../src/viper/lib/array.vpr")) ]
```

Comment:
This is temporary, right? We need to decide how to ship the viper lib with `moc` (probably ask the Dfinity team)

Reply:
Right, it's temporary. But I think it should be in this shape.
We will discuss with Dfinity team

Reply:
Tweaked path since it wasn't working for the test runner.
Now `lib/` should be next to `.mo` file.